src/UI/Component/Input/Field/Checkbox.php

```diff
@@ -11,23 +11,6 @@
 /**
  * This describes checkbox inputs.
  */
-interface Checkbox extends Group {
+interface Checkbox extends Input {
```

Comment:
Shouldn't this be `DependantGroupProviding` as well?

Reply:
I like the way the class-declation looks:
  class Checkbox extends Input implements C\Input\Field\Checkbox, C\Changeable, C\Onloadable, C\Input\Field\DependantGroupProviding 

with 
  interface Checkbox extends DependantGroupProviding 
we would have to stick to
  class Checkbox extends Input implements C\Input\Field\Checkbox, C\Changeable, C\Onloadable 
...which is fine, but - imo - not as nice.

Reply:
But this would also mean, that this is something that only is available internally, users of the UI-Framework won't see this. This of course is also true for `Changeable` and `Onloadable`. Is this what we want?